dev-libs/sway/metadata.xml

```diff
@@ -22,19 +26,29 @@
 		something else.
 	</longdescription>
 	<use>
-		<flag name="gdk-pixbuf">
-			Enable additional image format support via the
-			<pkg>x11-libs/gdk-pixbuf</pkg> library.
-		</flag>
-		<flag name="swaybar">Build 'swaybar': sway's status bar component.</flag>
-		<flag name="swaybg">Build 'swaybg': allows to set a desktop background image.</flag>
-		<flag name="swaygrab">Build 'swaygrab': allows taking screenshots or screencasts.</flag>
-		<flag name="swaylock">Build 'swaylock': sway's screen locker.</flag>
-		<flag name="swaymsg">Build 'swaymsg': tool for communicating with a running sway instance.</flag>
-		<flag name="wallpapers">Install sway's default wallpaper image.</flag>
-		<flag name="tray">Enable support for StatusNotifierItem tray specification.</flag>
+		<flag name="bash-completion" restrict="&gt;=dev-libs/sway-1.0_beta1">Enable bash completion support</flag>
+		<flag name="dmenu" restrict="&gt;=dev-libs/sway-1.0_beta1">Pull in <pkg>x11-misc/dmenu</pkg></flag>
+		<flag name="elogind" restrict="&gt;=dev-libs/sway-1.0_beta1">Enable support for rootless session via elogind</flag>
+		<flag name="filecaps" restrict="&gt;=dev-libs/sway-1.0_beta1">Use Linux capability SYS_CAP_ADMIN rather than set*id for sway binary</flag>
+		<flag name="fish-completion" restrict="&gt;=dev-libs/sway-1.0_beta1">Enable fish completion support</flag>
+		<flag name="gdk-pixbuf" restrict="&lt;dev-libs/sway-1.0_beta1">Enable additional image format support via the <pkg>x11-libs/gdk-pixbuf</pkg> library</flag>
+		<flag name="swaybar">Install 'swaybar': sway's status bar component</flag>
+		<flag name="swaybg">Install 'swaybg': allows to set a desktop background image</flag>
+		<flag name="swaygrab" restrict="&lt;dev-libs/sway-1.0_beta1">Install 'swaygrab': allows taking screenshots or screencasts</flag>
+		<flag name="swaylock">Install 'swaylock': sway's screen locker</flag>
+		<flag name="swaymsg" restrict="&lt;dev-libs/sway-1.0_beta1">Install 'swaymsg': tool for communicating with a running sway instance</flag>
+		<flag name="tray">Enable support for StatusNotifierItem tray specification</flag>
+		<flag name="wallpapers">Install sway's default wallpaper image</flag>
+		<flag name="xwayland">Enable support for X11 applications</flag>
```

Comment:
I'm unconvinced why this should be `xwayland` rather than `X`.

Reply:
Maybe @gentoo/x11 has a policy.

Reply:
I think X is better than Xwayland, yeah. I would indeed change weston and wlc as well. (My rationale is that X is the thing you want support for. Xwayland just happens to be how its implemented)

Reply:
Changed to `X` both here and in wlroots.